tests/test_wordpress_api.py

```diff
@@ -1,20 +1,13 @@
 import unittest
+import vcr
 
-from datetime import timedelta, date
+from datetime import timedelta, datetime
 from unittest.mock import patch
 from canonicalwebteam.blog import wordpress_api as api
+from canonicalwebteam.http import CachedSession
 
-
-class MockResponse:
-    ok = True
-
-    def json(self):
-        return ["hello_test"]
-
-    def raise_for_status(*args):
-        pass
-
-    headers = {"X-WP-TotalPages": 12}
+# make sure that timeouts do not hinder tests
+api.api_session = CachedSession(fallback_cache_duration=3600, timeout=(1, 100))
```

Comment:
Same comment. Replace with Session instead of using cache

Reply:
same comment. Don't touch Session if you want to keep the same behaviour than prod.